terumet/material/coalproc.lua

```diff
@@ -66,7 +66,7 @@ minetest.register_craftitem( rsuit_mat, {
     inventory_image = terumet.tex(rsuit_mat)
 })
 
-terumet.register_alloy_recipe{result=rsuit_mat, flux=8, time=10.0, input={rubber_bar_id, 'terumet:ingot_cgls'}}
+terumet.register_alloy_recipe{result=rsuit_mat, flux=8, time=20.0, input={rubber_bar_id, 'terumet:item_thermese', 'terumet:item_ceramic'}}
```

Comment:
Is there a particular reason you changed the recipe for vulcansuit plates?

Reply:
Oh, I meant to add an addendum about my other changes at the bottom of the original message, but forgot. Sorry. The vulcansuit felt like it was intended to be a fire-protection suit; i changed its properties and recipe accordingly. Teruceramic (insulating) plus Thermese (conducting) felt like a good combo for a heat-protection suit. I increased the time because its very powerful. I have no issue reverting this change if you disagree.

Reply:
The other major comment I had intended to make, was that the "use" values for the armors seem to have gotten inverted; the armors which were previously strong were made weak (use-wise), and vice versa. "uses" indicates the amount of wear the armor takes on each hit, out of 65535.

Reply:
I also added some spaces and re-ordered some things just to make it easier to see what I was doing as I fiddled several values; I can put back the original order/spacing if you wish.

Reply:
No problem! The original idea was to indeed that it was supposed to be a sort of high-end, powerful armor in general.

That also reminds me I had been meaning to add a small fire protection bonus to it (and the other armors) if only to offset the annoying torch damage for people, so thanks for doing that too!

The time increase is probably a good idea, really, and I can see what you're thinking for sure. While the rubber does require a high-end machine to create, it's not particularly expensive and I wanted to offset that with something of higher value and "hardness". Can you change the thermese to a coreglass ingot? I think that'd be pretty good.

Reply:
> The other major comment I had intended to make, was that the "use" values for the armors seem to have gotten inverted; the armors which were previously strong were made weak (use-wise), and vice versa. "uses" indicates the amount of wear the armor takes on each hit, out of 65535.

ARGH really?? I had no idea. That's obnoxious. I might just have the registration function do 65535/uses and assign that to the armor instead then.

Additionally I was just looking at how fire resistance actually works. I had just set the bracers to 99 since I figured that was high enough for immunity and it works fine; but it looks like the highest you really need for lava is a total of 8:

`	fire_nodes = {
		{"default:lava_source",     5, 8},
		{"default:lava_flowing",    5, 8},
		{"fire:basic_flame",        3, 4},
		{"fire:permanent_flame",    3, 4},
		{"ethereal:crystal_spike",  2, 1},
		{"ethereal:fire_flower",    2, 1},
		{"default:torch",           1, 1},
		{"default:torch_ceiling",   1, 1},
		{"default:torch_wall",      1, 1},
	},`

You may have seen this in 3d_armor's api.lua, but that 3rd element is the level of fire protection needed for resistance to that fire node